app/addons/documents/rev-browser/actions.js

```diff
@@ -63,81 +57,77 @@ function getConflictingRevs (paths, winner, deleted) {
   });
 }
 
-function dispatchData (tree, doc, conflictingRevs, conflictDoc, databaseName) {
-  FauxtonAPI.dispatch({
+const dispatchData = (tree, doc, conflictingRevs, conflictDoc, databaseName) => {
```

Comment:
To avoid confusion, I'd rename this to `treeLoaded` or `dataLoaded` since it's not longer calling `dispatch()`.